### ISSUE TITLE
EDDiscovery crash while closing the app

### ISSUE THREAD

**Issue by xenophonf** [EDDiscovery/EDDiscovery#2876]
I stopped hearing EDDiscovery voice in-game events after a while.  I clicked the window close button, and it printed the "Closing - please wait" message in the status bar at the bottom of the EDDiscovery main window.  After about 10-15 minutes, I clicked the window close button again, when EDDiscovery crashed:

```
An unhandled fatal exception has occurred.

==== BEGIN ====
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'EDDiscoveryForm'.
   at System.Windows.Forms.Control.MarshaledInvoke(Control caller, Delegate method, Object[] args, Boolean synchronous)
   at System.Windows.Forms.Control.Invoke(Delegate method, Object[] args)
   at System.IO.FileSystemWatcher.OnCreated(FileSystemEventArgs e)
   at System.IO.FileSystemWatcher.CompletionStatusChanged(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pOVERLAP)
===== END =====
```

This has happened before, where EDDiscovery will stop responding after running constantly for about a day.

**Comment by robbyxp1** [comment]
This may just be a random winforms incident. Sometimes the underlying system does not close down properly.

Could you paste in some logs, found in %LOCALAPPDATA%/eddiscovery/logs, which have the exceptions listed. Then we can see if another exception earlier on caused it.

**Comment by klightspeed** [comment]
The screenshot watcher isn't being stopped when the program shuts down.

**Comment by robbyxp1** [comment]
We have fixed this on 11.6, thanks for the report!
